projects.js

```diff
@@ -0,0 +1,206 @@
+          >${technologies[2]}</a
+        >
+      </li>
+    </ul>
+    <button
+      class="inline-block open-popup-btn transition duration-500 ease-in-out text-primary-blue p-4 text-[17px] border border-light-blue rounded-lg mb-[14px] md:enabled:text-light-blue md:enabled:border-[#7F8CFF] md:enabled:bg-white md:hover:bg-light-blue md:hover:text-white md:active:bg-[#2230D2] md:active:text-white md:disabled:bg-[#e5e5e5] md:disabled:text-[#5E6C84] md:disabled:border-[#C1C7D0]"
+      >${button}</button
+    >
+  </div>
+
+  <!-- modal -->
+  <div class="z-20 hidden modal-wrapper">
+    <div class="modal-content bg-white z-10 rounded-2xl py-[18px] px-4">
+      <header class='flex justify-between items-center'>
+        <h3 class="font-bold text-[32px] self-end">${name}</h3>
+        <button class='close-popup'>
+          <img class="h-[12px] w-[12px]" src='./img/close_3.png' alt='close button'/>
+        </button>
+      </header>
+
+      <!-- modal subtitle -->
+      <div class="text-[13px] font-semibold mt-[22px] flex justify-between items-center w-[75%] md:w-[260px] text-light-grey-4">
+        <span class="uppercase text-[#344563]">${subtitle[0]}</span>
+        <span class="h-[10px] w-[10px] bg-light-grey rounded-full"></span>
+        <span class="capitalize">${subtitle[1]}</span>
+        <span class="h-[10px] w-[10px] rounded-full bg-light-grey"></span>
+        <span>${subtitle[2]}</span>
+      </div>
+
+      <div class="image-wrapper mt-[20px]">
+        <img class="object-cover" src=${featuredImage} alt="project image"/>
+      </div>
+
+      <div class="modal__bottom-section md:flex">
+        <div class="text-wrapper">
+          <p class="mt-[12px] text-[15px] text-[#344563] leading-6">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a galley of type and scrambled it to make a type 
+            specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+          </p>
+        </div>
+
+        <div class="modal__bottom-left-section">
+          <div class="mt-[12px]">
+            <span class="text-light-blue text-xs px-3 py-1 bg-faded-blue rounded-lg">${technologies[0]}</span>
+            <span class="text-light-blue ml-[8px] text-xs px-3 py-1 bg-faded-blue rounded-lg">${technologies[1]}</span>
+            <span class="text-light-blue ml-[8px] text-xs px-3 py-1 bg-faded-blue rounded-lg">${technologies[2]}</span>
+          </div>
+
+          <!-- modal buttons -->
+          <ul class="flex mt-[49px] modal-buttons">
+            <li class="flex justify-between md:cursor-pointer items-center rounded-lg border border-[#6070FF] py-[14px] px-[22px]">
+              <span class="text-[15px] text-[#396DF2] font-medium">See live</span>
```

Comment:
Kindly use the `a` element instead of the `span` element and store our live link in your `projectsList` array as requested [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=link%20to%20live%20version).